lib/edit-site-export.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * REST endpoint for exporting the contents of the Edit Site Page editor.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Output a ZIP file with an export of the current templates
+ * and template parts from the site editor, and close the connection.
+ */
+function edit_site_export() {
```

Comment:
Shouldn't this be prefixed with `gutenberg`?

Reply:
Done